var/spack/repos/builtin/packages/root/package.py

```diff
@@ -31,6 +31,8 @@ class Root(CMakePackage):
     # Development version (when more recent than production).
 
     # Production version
+    version("6.26.10", sha256="8e56bec397104017aa54f9eb554de7a1a134474fe0b3bb0f43a70fc4fabd625f")
+    version("6.26.08", sha256="4dda043e7918b40743ad0299ddd8d526b7078f0a3822fd06066df948af47940e")
```

Comment:
If I understand the release notes correctly, [you need 6.26.08+ to build with nlohmann_json 3.11+](https://github.com/root-project/root/issues/11312). The dependency on nlohmann_json should probably be updated accordingly.

Reply:
Similarly, clang 16 will have issues with older root : https://github.com/root-project/root/issues/11128 .